src/create-twilio-function/create-files.js

```diff
@@ -19,19 +19,38 @@ async function createFile(fullPath, content) {
   return writeFile(fullPath, content, { flag: 'wx' });
 }
 
-function createPackageJSON(pathName, name) {
+const javaScriptDeps = {};
+const typescriptDeps = { '@twilio-labs/serverless-runtime-types': versions.serverlessRuntimeTypes };
+const javaScriptDevDeps = { 'twilio-run': versions.twilioRun };
+const typescriptDevDeps = {
+  'twilio-run': versions.twilioRun,
+  typescript: versions.typescript,
+  copyfiles: versions.copyfiles,
+};
+
+function createPackageJSON(pathName, name, projectType = 'javascript') {
   const fullPath = path.join(pathName, 'package.json');
+  const scripts = {
+    test: 'echo "Error: no test specified" && exit 1',
+    start: 'twilio-run',
+    deploy: 'twilio-run deploy',
+  };
+  if (projectType === 'typescript') {
+    scripts.build = 'tsc && npm run build:copy-assets';
+    scripts['build:copy-assets'] = 'copyfiles src/assets/* src/assets/**/* --up 2 --exclude **/*.ts dist/assets/';
+    scripts.prestart = 'npm run build';
+    scripts.predeploy = 'npm run build';
+    scripts.start += ' --cwd dist --env ../.env';
```

Comment:
I have a PR open that adds the respective flags so you don't need to use the --cwd

Reply:
I shall update!